### PR TITLE
C++ <complex> Number Cheat Sheet

### PR DESCRIPTION
**Pull request by ManrajGrover**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {C++ Complex Numbers} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

This pull request adds C++ <complex> library cheat sheet for reference.

###### Which issues (if any) does this fix?

None

###### People to notify (@mention interested parties)

@zekiel @moollaza 
---

Instant Answer Page: https://duck.co/ia/view/cpp_complex_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention
None

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/cpp-complex.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
+{
+    "id": "cpp_complex_cheat_sheet",
+    "name": "C++ Complex Numbers",
+    "description": "C++ <complex> library implements the complex class to contain complex numbers in cartesian form and several functions and overloads to operate with them.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/cpp-complex.json]: remove period at end of description

Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/cpp-complex.json]: Made a quick change. Please check.

**Comment by zekiel** [comment]
LGTM!

**Pull request merged**